test/TestUtilities/Test.Utility/ResourceTestUtility.cs

```diff
@@ -26,6 +26,8 @@ public static void CopyResourceToFile(string resourceName, Type type, string fil
 
         public static byte[] GetResourceBytes(string name, Type type)
         {
+            var x = type.GetTypeInfo().Assembly.GetManifestResourceNames();
```

Comment:
why do you need this? should you just remove it?

Reply:
yup. missed that one! Thanks!